test/python/algorithms/optimizers/test_optimizer_aqgd.py

```diff
@@ -14,14 +14,20 @@
 
 import unittest
 from test.python.algorithms import QiskitAlgorithmsTestCase
-from qiskit import Aer
 from qiskit.circuit.library import RealAmplitudes
 from qiskit.utils import QuantumInstance, algorithm_globals
 from qiskit.opflow import PauliSumOp
 from qiskit.algorithms.optimizers import AQGD
 from qiskit.algorithms import VQE, AlgorithmError
 from qiskit.opflow.gradients import Gradient
 from qiskit.test import slow_test
+from qiskit.utils import optionals
+
+if optionals.HAS_AER:
+    # pylint: disable=import-error,no-name-in-module
+    from qiskit_aer import Aer  # pylint: disable=import-error
```

Comment:
Are we leaving the tests as requiring Aer 0.11+?  I don't think I have a massive problem with it, just pointing it out.

Reply:
Yeah, this was intentional, for tests I didn't see a point in continuing to support the old versions